README.md

```diff
@@ -62,7 +62,7 @@ npm install
 * Populate MongoDB with seed data required for the app
     * Run grunt task below to populate the DB with seed data required for the application. Pass the desired environment as argument. If not passed, "development" is the default:
 ```
```

Comment:
Please verify passing environment as arguments work with npm script. Once confirmed, please add and example syntax showing how to pass  it.